### PR TITLE
Recalculate calculated fields on edit

### LINKED ISSUE #159: Calculated fields are not recalculated when location is edited
**Describe the bug**
@mzagaja Calculated fields are not recalculated if a development is edited.

**Steps to reproduce**
1. Open a development that passes validations
2. Query for that development's calculated fields in psql
3. Edit the development such that it is located in a new RPA region
4. Query for that development's calculated fields again
5. Notice that the rpa_name, county, n_transit, neighborhood, and loc_id have not changed.

**Expected behavior**
rpa_name, county, n_transit, neighborhood, and loc_id should update when a development's location has changed.

### PR DESCRIPTION
**Pull request by iangmhill**
Resolves #159

### CODE REVIEW

File: ember/app/components/mapbox-map.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ import centerOfMass from 'npm:@turf/center-of-mass';
 
 mapboxgl.accessToken = 'pk.eyJ1IjoiaWhpbGwiLCJhIjoiY2plZzUwMTRzMW45NjJxb2R2Z2thOWF1YiJ9.szIAeMS4c9YTgNsJeG36gg';
```
Comment (mzagaja) [inline ember/app/components/mapbox-map.js]: Access tokens for APIs generally should not be checked into GitHub repos. Typically we use environment variables or secrets.yml to obscure these. Unless it is a public access token that is ok to expose to the public (not tied to our account limits, etc.)

**Comment by mzagaja** [comment]
It appears that this bug reproduces if you submit an edit that does not include a change to lat/long to Dunkin Donuts: http://127.0.0.1:4200/map/developments/3103/edit

**Comment by iangmhill** [comment]
@mzagaja Run `rake database:populate_long_lat` to populate the latitude and longitude columns and everything should work.

**Comment by mzagaja** [comment]
Ok that looks like it fixed it! 👍🏼

**Comment by mzagaja** [comment]
I would just add a note about this task to README.md, however.

**Pull request merged**